docs/api/schemas.md

```diff
@@ -4035,3 +4377,54 @@ It corresponds to the legacy derpN.tailscale.com hostnames used by older clients
 RegionIDs must be non-zero, positive, and guaranteed to fit in a JavaScript number.
 RegionIDs in range 900-999 are reserved for end users to run their own DERP nodes.|
 |`regionName`|string|false||Regionname is a long English name for the region: "New York City", "San Francisco", "Singapore", "Frankfurt", etc.|
+
+## time.Duration
+
+```json
+-9223372036854776000
+```
+
+### Properties
+
+| Name          | Type    | Required | Restrictions | Description |
+| ------------- | ------- | -------- | ------------ | ----------- |
+| `*anonymous*` | integer | false    |              |             |
+
+#### Enumerated Values
+
+| Property      | Value                  |
```

Comment:
A lot of noise here...

Reply:
Oh, I see what's going on. It's interpreting `time.Duration` as an enum. Maybe we update `.swaggo` with `replace time.Duration int64`

Reply:
Fixed